capstone/src/main/resources/static/script.js

```diff
@@ -462,17 +462,46 @@ function searchOrg() {
 /* function to generate divs for the calendar */
 function createCalendar() {
   const calendar = document.getElementById("calendar");
-  for (var i = 0; i < 14; i++) {
+
+  var today = new Date();
+  var endDate = new Date();
+  endDate.setDate(today.getDate() + 8);
+  for (var i = 0; i < 7; i++) {
     var nextDay = new Date();
-    var today = new Date();
     nextDay.setDate(today.getDate() + i);
     const dateDiv = document.createElement('div');
     dateDiv.setAttribute("class", "date general-container");
     const dateDisplay = document.createElement('p');
     dateDisplay.innerText = nextDay.toDateString();
     dateDiv.appendChild(dateDisplay);
+    const eventDiv = document.createElement('div');
+    eventDiv.setAttribute("class", "date row");
+    eventDiv.setAttribute("id", "date" + i);
     calendar.append(dateDiv);
+    calendar.append(eventDiv);
   }
+  fetch('get-calendar-events').then(response => response.json()).then((data) => {
```

Comment:
Does this add all events?

Reply:
Yes for now. This is because I made the change into references in the other PR so until that PR is merged I don't want to change it on here.